README.md

```diff
@@ -1,23 +1,68 @@
-# Next.js + Jest
+# Sparkbox Apprentice Page
 
-This example shows how to configure Jest to work with Next.js.
+## Contributing 
 
-This includes Next.js' built-in support for Global CSS, CSS Modules, and TypeScript!
+##### Git:
 
-## How to Use
+- When you want to make a contribution please follow the [Sparkbox git process](https://github.com/sparkbox/standard/tree/main/code-style/git) for naming branches, writing commit messages, and rebasing before submitting PRs.
 
-Quickly get started using [Create Next App](https://github.com/vercel/next.js/tree/canary/packages/create-next-app#readme)!
+##### Updating Apprentices Data:
 
-In your terminal, run the following command:
+  ###### Images 
 
-```bash
-npx create-next-app --example with-jest with-jest-app
-# or
-yarn create next-app --example with-jest with-jest-app
-```
+  - All image files should be added to ```/public/apprentices``` with the file name being ```[ApprenticeName].png```
 
-## Run Jest Tests
+  ###### Apprentice Data
 
-```bash
-npm test
-```
+  - To add/change the current apprentices or previous apprentices data navigate to ```/data``` folder and find the ```apprentices.json``` file
+
+  - Add to and/or adjust the data, see below how the data for each group is structured:
+    - *Current Apprentices*:
+    ```js
+    "currentApprentices": {
+      "version": "[Version Number]",
+      "apprentices": [
+        ...,
+        {
+          "name": "[Apprentice Name]",
+          "image": "/apprentices/[Name].png",
+          /* Links are optional and can be any link the apprentice provides as long as it follows 
+          the structure of the objects below: */
+          "links": [
+            { "href": "https://www.personalportfolio.com", "text": "Personal" },
+            { "href": "https://www.linkedin.com/", "text": "Linkedin" },
+            { "href": "https://www.github.com/", "text": "Github" },
+          ],
+        },
+      ],
+    },
+     ```
+    - *Previous Apprentices*:
+    ```js 
+    "previousApprentices": [
+      ...,
+      {
+        "version": "[Version Number]",
+        "apprentices": [
+          ...,
+          {
+            "name": "[Apprentice Name]",
+            /* Status is optional and states the apprentice employment status with Sparkbox
+            and can be one of the following values, if the apprentice was never an employee omit
+            status entirely: */
+            "status": [ "current" || "previous" ]
+          }
+        ]
+      }
+    ]
+    ```
+
+  ##### Updating Snapshot: 
+    
+  - If any markup is changed the snapshot will need to be updated to represent those changes in the test
+  - run ```npm test``` and then hit ```u``` to update the snapshot when the test fails 
```

Comment:
Please update this to include the [npm script](https://github.com/sparkbox/apprentices-page-next/blob/main/package.json#L13) that Alice added: `npm run test:snapshot:update`. Looks great otherwise! Thanks!